src/buildercore/command.py

```diff
@@ -20,23 +35,41 @@
 class CommandException(Exception):
     pass
 
-# no un-catchable errors from Fabric
-env.abort_exception = CommandException
+if BACKEND == FABRIC:
+    # no un-catchable errors from Fabric
+    fab_api.env['abort_exception'] = CommandException
+else:
+    threadbare.state.set_defaults({"abort_exception": CommandException})
 
 NetworkError = fab_exceptions.NetworkError
 
 #
 # api
 #
 
-local = fab_api.local
-execute = fab_api.execute
-parallel = fab_api.parallel
-serial = fab_api.serial
-hide = fab_api.hide
+def fab_api_local_remote_wrapper(fab_result):
```

Comment:
the name is a bit confusing given there is also `fab_api_local_wrapper`; this is one layer down from other `fab_api_*_wrapper` functions

Reply:
I've renamed if `fab_api_results_wrapper`, converted it to a decorator and removed the intermediate functions.